src/components/map/Options.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+import styled from "styled-components";
+
+import Modal from "../common/Modal";
+
+const Option = ({ children: option }) => {
+  const navigate = useNavigate();
+  const { search, pathname } = useLocation();
+
+  const navigatePage = destination => {
+    const path = destination === "수정" ? "photo-editor" : "new-photo";
+
+    const url = pathname.concat(`/${path}${search}`);
+    navigate(url);
```

Comment:
```suggestion
    const path = destination === "수정" ? "photo-editor" : "new-photo";
    const url = pathname.concat(`/${path}${search}`);
    
    navigate(url);
```

위에 붙이고 아래를 개행하는건 어떻게 생각하시나요-??

Reply:
좋은 것 같습니당 🙌

Reply:
좋습니다 :)